_pages/get_involved.md

```diff
@@ -0,0 +1,14 @@
+---
+---
+
+# Get involved in making BIDS better
+
+You can contribute to the BIDS specification by opening [Issues](https://github.com/bids-standard/bids-specification/issues) and propose changes via [Pull Requests](https://github.com/bids-standard/bids-specification/pulls).
```

Comment:
```suggestion
The easiest way to contribute to BIDS is to ask questions you have about the specification on [Neurostars](https://neurostars.org.)
If your question has a [bids tag](https://neurostars.org/search?q=tags%3Abids) it will be much easier for others to find the answer.

You can also get involved by _answering_ questions on [Neurostars](https://neurostars.org/search?q=tags%3Abids)!

You can contribute to the BIDS specification by opening [Issues](https://github.com/bids-standard/bids-specification/issues) and propose changes via [Pull Requests](https://github.com/bids-standard/bids-specification/pulls) on [GitHub](https://github.com/bids-standard/bids-specification).
```